package.json

```diff
@@ -9,17 +9,19 @@
     "helloworld": "asc --transform json-as/transform assembly/helloworld.ts --target release --exportRuntime --textFile build/helloworld.wat --binaryFile build/helloworld.wasm",
     "tokenspender": "asc --transform json-as/transform assembly/tokenspender.ts --target release --exportRuntime --textFile build/tokenspender.wat --binaryFile build/tokenspender.wasm",
     "create_token": "node massa_tools/include_base64.js assembly/create_token.ts && asc --transform json-as/transform assembly/create_token.m.ts --target release --exportRuntime --textFile build/create_token.wat --binaryFile build/create_token.wasm && rm assembly/create_token.m.ts",
-    "test_tokenspend": "node massa_tools/include_base64.js assembly/test_tokenspend.ts && asc --transform json-as/transform assembly/test_tokenspend.m.ts --target release --exportRuntime --textFile build/test_tokenspend.wat --binaryFile build/test_tokenspend.wasm && rm assembly/test_tokenspend.m.ts"
+    "test_tokenspend": "node massa_tools/include_base64.js assembly/test_tokenspend.ts && asc --transform json-as/transform assembly/test_tokenspend.m.ts --target release --exportRuntime --textFile build/test_tokenspend.wat --binaryFile build/test_tokenspend.wasm && rm assembly/test_tokenspend.m.ts",
+    "receive_message": "asc assembly/receive_message.ts --target release --exportRuntime --binaryFile build/receive_message.wasm",
+    "send_message": "yarn run receive_message && node massa_tools/include_base64.js assembly/send_message.ts && asc assembly/send_message.m.ts --target release --exportRuntime --textFile build/send_message.wat --binaryFile build/send_message.wasm && rm assembly/send_message.m.ts"
   },
   "devDependencies": {
-    "assemblyscript": "^0.19.20",
+    "assemblyscript": "^0.19.23",
     "jest": "^27.4.5"
   },
   "dependencies": {
     "as-base58": "^0.1.1",
     "as-hmac-sha2": "^1.0.2",
     "json-as": "^0.2.6",
-    "massa-sc-std": "https://github.com/z80dev/massa-sc-std",
+    "massa-sc-std": "https://github.com/massalabs/massa-sc-std#5c90b49",
```

Comment:
I don't think you want to use a specific branch. Let's just wait for the `massa-sc-std` branch with the `send_message` to be merged.